EasyPost/Constants.cs

```diff
@@ -90,5 +90,17 @@ public static class ErrorMessages
             internal const string ApiErrorDetailsParsingError = "RESPONSE.PARSE_ERROR"; // not for public consumption
             public const string CouldNotPassClient = "Could not pass client to {0}.";
         }
+
+        public static class CarrierAccountTypes
```

Comment:
Question: Do we want to separate normal constants and error strings to separate files?

Reply:
Having two constants files seems redundant and counter-intuitive to me.